browser/ui/BUILD.gn

```diff
@@ -479,17 +479,22 @@ source_set("ui") {
     sources += [
       "views/toolbar/wallet_button.cc",
       "views/toolbar/wallet_button.h",
+      "webui/brave_wallet/deprecated_brave_wallet_ui.cc",
+      "webui/brave_wallet/deprecated_brave_wallet_ui.h",
+      "webui/brave_wallet/wallet_page_ui.cc",
+      "webui/brave_wallet/wallet_page_ui.h",
       "webui/brave_wallet/wallet_panel_ui.cc",
       "webui/brave_wallet/wallet_panel_ui.h",
-      "webui/brave_wallet_ui.cc",
-      "webui/brave_wallet_ui.h",
     ]
     deps += [
+      "webui/brave_wallet/common_handler",
       "webui/brave_wallet/page_handler",
-      "//brave/components/brave_wallet_ui:brave_wallet_page_generated",
-      "//brave/components/brave_wallet_ui:brave_wallet_panel_generated",
+      "webui/brave_wallet/panel_handler",
```

Comment:
leading `//`

Reply:
for a relative path?

Reply:
yes, paths should be absolute

Reply:
Is there some documentation that says that? I see relative paths in BUILD.gn files very commonly when they are subdirectories.

Reply:
https://gn.googlesource.com/gn/+/refs/heads/main/docs/style_guide.md#deps

Reply:
ok thanks, I'll update in a follow up PR.